include/nuttx/can/sja1000.h

```diff
@@ -155,7 +155,7 @@ struct sja1000_dev_s
  ****************************************************************************/
 
 static inline uint32_t sja1000_getreg(struct sja1000_dev_s *sja_priv,
```

Comment:
```suggestion
static inline uint32_t sja1000_getreg(FAR struct sja1000_dev_s *sja_priv,
```

Reply:
drop this change since it's added by accident.